docs/connector-development/cdk-python/schemas.md

```diff
@@ -25,3 +25,85 @@ def get_json_schema(self):
     return schema
 ```
 
+## Schema normalization
+
+It is important to ensure output data conforms to the declared json schema. This is because the destination receiving this data to load into tables may strictly enforce schema (e.g. when data is stored in a SQL database, you can't put CHAT type into INTEGER column). In the case of changes to API output (which is almost guaranteed to happen over time) or a minor mistake in jsonschema definition, data syncs could thus break because of mismatched datatype schemas.
```

Comment:
These docs are fantastic! very thorough and user friendly.

Reply:
Thanks!